web/client/codechecker_client/credential_manager.py

```diff
@@ -64,33 +64,45 @@ def __init__(self):
         user_home = os.path.expanduser("~")
         session_cfg_file = os.path.join(user_home,
                                         ".codechecker.passwords.json")
-        LOG.debug(session_cfg_file)
+        LOG.info("Checking local passwords or tokens in %s", session_cfg_file)
 
         scfg_dict = {}
         if os.path.exists(session_cfg_file):
+            check_file_owner_rw(session_cfg_file)
             scfg_dict = load_json_or_empty(session_cfg_file, {},
                                            "user authentication")
             scfg_dict['credentials'] = \
                 simplify_credentials(scfg_dict['credentials'])
-        if os.path.exists(session_cfg_file):
-            check_file_owner_rw(session_cfg_file)
+
+            if not scfg_dict['credentials']:
+                LOG.info("No saved tokens.")
+            else:
+                LOG.debug("Tokens or passwords were found for these hosts:")
+                for k, v in scfg_dict['credentials'].items():
+                    user, _ = v.split(":")
+                    LOG.debug("  user '%s' host '%s'", user, k)
+
+        else:
+            LOG.info("Password file not found, checking for valid sessions")
 
         if not scfg_dict.get('credentials'):
             scfg_dict['credentials'] = {}
 
         self.__save = scfg_dict
         self.__autologin = scfg_dict.get('client_autologin', True)
-
         # Check and load token storage for user.
         self.token_file = os.path.join(user_home, ".codechecker.session.json")
-        LOG.debug(self.token_file)
+        LOG.info("Checking for local valid sessions.")
 
         if os.path.exists(self.token_file):
             token_dict = load_json_or_empty(self.token_file, {},
                                             "user authentication")
             check_file_owner_rw(self.token_file)
 
             self.__tokens = token_dict.get('tokens')
+            LOG.info("Found session information for these hosts:")
+            for k, _ in self.__tokens.items():
+                LOG.debug("  %s", k)
```

Comment:
I think these messages should go to the same log level. Otherwise the `info` level will not tell what is the found session information after the colon chatacter.